<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>driverUI</class>
 <widget class="QWidget" name="driverUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1183</width>
    <height>674</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="viewDeliveries">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>80</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>View Deliveries</string>
   </property>
  </widget>
  <widget class="QPushButton" name="editDetails">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>80</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Edit Details</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>400</y>
     <width>261</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Available orders</string>
   </property>
  </widget>
  <widget class="QComboBox" name="avaliabilityCombo">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>20</y>
     <width>201</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>0</y>
     <width>201</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Set Availability</string>
   </property>
  </widget>
  <widget class="QPushButton" name="update_avaliability">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>20</y>
     <width>191</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Update</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>0</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Current Status:</string>
   </property>
  </widget>
  <widget class="QLabel" name="avaliabilityLabel">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>0</y>
     <width>161</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QTableWidget" name="new_deliveries">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>450</y>
     <width>1121</width>
     <height>181</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="connect_to_topic">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>80</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Connect to Topic</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="existing_orders">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>1121</width>
     <height>281</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>Customer Name</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Order ID</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Destination Postcode</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Distance</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Height(M)</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Length(M)</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Width(M)</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Weight(KG)</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Shpping Cost(Â£)</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Delivery Status</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Update</string>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>261</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Current Orders</string>
   </property>
  </widget>
  <widget class="QPushButton" name="sign_out">
   <property name="geometry">
    <rect>
     <x>1054</x>
     <y>0</y>
     <width>131</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Sign Out</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
